### DISCUSSION TITLE
Can't bridge Sapolia to Zksync

### DISCUSSION THREAD

**Discussion by Totiruzi** [Cyfrin/foundry-full-course-cu#2446] [General]
I am having error of not sufficeient fund when trying to bridge Sapolia to Zksync. This is the error I got
```Sapolia
Insufficient ETH balance on Ethereum Sepolia Testnet to cover the fee. We recommend having at least 0.004420299427175108 ETH on Ethereum Sepolia Testnet for deposit.
```
these are screenshots from trying to bridge

![Screenshot from 2024-09-11 14-12-17](https://github.com/user-attachments/assets/3e45b525-6308-428d-a683-297ae5e43451)
![Screenshot from 2024-09-11 14-12-31](https://github.com/user-attachments/assets/55a9ec76-72eb-43da-9204-5dba88ad54ca)

**Comment by Totiruzi** [comment]
As can be seen from the screenshot I have 0.05 Sapolia in my metamask
Thanks )

**Reply by EngrPips** [reply]
I did not see the `0.05` anywhere in the screenshot. Please point me to it.

**Reply by EngrPips** [reply]
I was able to successfully bride `0.3` sepolia eth to zksync sepolia eth.

<img width="1440" alt="Screenshot 2024-09-11 at 12 25 57" src="https://github.com/user-attachments/assets/cdfdf6d0-b784-4908-a4a5-07e47235e40f">

<img width="1440" alt="Screenshot 2024-09-11 at 12 25 34" src="https://github.com/user-attachments/assets/7daa42bf-ea84-45cb-a2fc-ead24e25bcbc">

Are you sure you have `0.5` sepolia eth in your wallet?

**Reply by Totiruzi** [reply]
The second screenshot

**Comment by Totiruzi** [comment]
I just took a screenshot of my balance in metamask 

![image](https://github.com/user-attachments/assets/1d3634c4-3534-46d9-b93b-bd76a71ea19a)

**Reply by EngrPips** [reply]
It seems weird that you have the balance. Maybe refresh and retry. I was able to bridge 0.3 sepolia eth successfully
<img width="1440" alt="Screenshot 2024-09-11 at 14 17 51" src="https://github.com/user-attachments/assets/ba711a99-d561-4f9e-8c42-44dc5df2d49c">

**Reply by Totiruzi** [reply]
Thanks for your responses, I have ben trying all day. Will try tomorrow again. It's wired to me too. By the way I am on Fedora 40 and I tried using both Chrome and Firefox but got the same results.

Will keep this trend posted if I finally make the swap.
Thanks )

**Reply by EngrPips** [reply]
I am looking forward to the outcome when you retry.

**Reply by AndreaT95** [reply]
I'm having the same issue

**Reply by EngrPips** [reply]
I am not sure why you people are running into this issue. I didn't when I bridge yesterday.

**Reply by Totiruzi** [reply]
I have been trying all day, still can't bridge. Tried both Chrome and Firefox (

**Reply by miguelfaldutti87** [reply]
I'm having the same issue :(
I dont know what's happening.....

**Reply by EngrPips** [reply]
I will try do another bridging now and see if works, I am tired of hearing this issue and everything is working for me lol.

**Reply by EngrPips** [reply]
So i bridge another `0.1` sepolia eth to zksync sepolia eth. What are you guys doing wrongly? 🤔

<img width="1440" alt="Screenshot 2024-09-12 at 21 20 50" src="https://github.com/user-attachments/assets/04cbe9f0-a2c5-4058-a619-2b0a46f0744b">

<img width="1440" alt="Screenshot 2024-09-12 at 21 20 58" src="https://github.com/user-attachments/assets/d67fa7a2-7172-4c04-b4ea-c4812c7e9f00">

<img width="1440" alt="Screenshot 2024-09-12 at 21 21 25" src="https://github.com/user-attachments/assets/6d6ca971-5a18-4dcb-b042-b82c656b38ec">

**Comment by KevinLocksy** [comment]
Hi,
I encounter the same issue. Have you been able to solve it? @Totiruzi

**Comment by AlimGlck** [comment]
I run into the same issue also. I have 0.2984 eth and according to its warning, it shouldn't be a warning since I have enough faucets.

**Reply by miguelfaldutti87** [reply]
Not only that.... if you click on the coin selection button, you will see that the web recognize your 0.05 ETH.... but the error is still there.

**Reply by AlimGlck** [reply]
I just tried bridging on a different website and realized that when I connected my wallet and picked the Sepolia it showed my balance 0 although I have 0.3. Today I got some faucets for zkSync Sepolia Testnet and when I picked the zkSync Sepolia Testnet the faucets are shown on the website.  I guess the problem is with Sepolia Testnet in our case bc zkSync Sepolia Testnet works well. And if I want to bridge to the opposite (zkSync Sepolia to Etherium Sepolia) it is completely fine.

**Comment by EngrPips** [comment]
I didn't run into any problems.

**Comment by suvasis-patra** [comment]
I am having the same issue. Did anyone find a solution to the problem?

**Comment by Totiruzi** [comment]
Just a few minutes ago, I was able to bridge from Sepolia to zkSync after so many days of trying

**Reply by Totiruzi** [reply]
screenshot attached

![image](https://github.com/user-attachments/assets/6e333472-2f2f-4a9b-a902-7170f52b0327)

**Reply by EngrPips** [reply]
Finally, I am happy to know that it worked at last.

**Reply by AlimGlck** [reply]
Did you do anything different to make it work?

**Reply by Totiruzi** [reply]
I actually did not do anything differently.As usual restart my computer and tried the swap and it worked both on Chrome and Firefox. 
@KevinLocksy hope you got your working too!

**Reply by KevinLocksy** [reply]
I just checked and it also worked.

I did not do anything special neither.

**Comment by AlimGlck** [comment]
Okay, mine worked too. I didn't do anything special, but click the "change" button once.

![image](https://github.com/user-attachments/assets/59f4dce0-36c1-4592-9996-a2f35370ac7e)

**Comment by EngrPips** [comment]
Interesting

**Comment by miguelfaldutti87** [comment]
It worked for me too.
I didnt do anything either.... I just tried again and it worked... Maybe it was a temporal bug or something

**Comment by EngrPips** [comment]
I am glad it works for everybody now.

**Comment by dataneon** [comment]
I'm running into this problem now. 

The error states, ```Insufficient ETH balance on Ethereum Sepolia Testnet to cover the fee```, which I suppose makes sense, because I only have 0.4 Sepolia ETH and the current fee to bridge from Ethereum Sepolia Testnet to zkSync Sepolia Testnet is 1.36 Sepolia ETH.

Where can I get massive amounts of Sepolia ETH to do this bridge?

**Reply by EngrPips** [reply]
I will advise you to just leave the `zkSync` part as it is optional. getting massive sepolia isn't something easy lately.

**Reply by dataneon** [reply]
Okay, will do.

It looks like in the last few days there has been extraordinary traffic on Eth Sepolia, and so my problem might not have been the bridge, but the gas price.

Harry Papacharissiou of Chainlink Labs [posted yesterday on X](https://x.com/pappas9999/status/1843086535035588659) the following:

> In the past few days, Sepolia has become nearly unusable due to an unprecedented surge in blockspace demand. After investigating, I found some L2s running airdrop campaigns on Sepolia, rewarding users for bridging Sepolia ETH to their L2s and completing tasks.

**Reply by EngrPips** [reply]
Interesting.
